gcp-tracing/build.gradle

```diff
@@ -12,7 +12,11 @@ dependencies {
 
     implementation'io.zipkin.gcp:zipkin-sender-stackdriver:1.0.4'
     implementation 'io.zipkin.gcp:brave-propagation-stackdriver:1.0.4'
-    implementation 'io.grpc:grpc-auth:1.45.1'
+    implementation('io.grpc:grpc-auth:1.45.1'){
+        //grpc-auth pulls in an older version of google-auth-library-credentials
+        // which causes issues with maven native image compilation
```

Comment:
If it pulls in an older version, should we have a dependency to _upgrade_ it instead? Is that dependency actually redundant?

Reply:
Right, if we exclude it from `grpc-auth`, where is `google-auth-library-credentials` coming from then?

As Cédric says, maybe is better to explicitly add the dependency to a version that we know works well.

Reply:
The GCP bom has:
`api("com.google.auth:google-auth-library-oauth2-http:${googleAuthLibraryOauth2HttpVersion}")`
which pulls in `com.google.auth:google-auth-library-credentials` with the same version as the oauth2 version.

Reply:
But the BOM only applies if there's a dependency with the same coordinates, so it kind of works by accident. I think we should add a constraint on the `google-auth-library-credentials` library instead of using an exclude, which basically says "I don't need this", which isn't true.